.gitignore

```diff
@@ -1,3 +1,6 @@
 /dist/
 /gtk2hs_macros.h
 /dist-newstyle/
+.stack-work/
+*.cabal
```

Comment:
I am fairly certain we do not want to ignore the .cabal file.

Reply:
see my above comment